### PR TITLE
ci: new action for performing transform

### PR DESCRIPTION
**Pull request by jmgamboa**
### Jira Ticket(s)

- [PLAT-3867](https://leaflink.atlassian.net/browse/PLAT-3867)

CI step for installing, running, and outputing oast ingest-ci 

Screen Shot 2022-10-12 at 2 35 05 PM

### CODE REVIEW

File: frosting-openapi-upstream-transform/action.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ inputs:
     description: "Applies the defined global transformations to the oas document"
     required: true
     default: true
+  automation-pat:
+    description: 'A GitHub Personal Access Token'
+    required: true
```
Comment (shonteag) [inline frosting-openapi-upstream-transform/action.yml:10]: Yup, this should do the trick. Anything secret has to passed in, from experience.

**Review by TNAucoin — APPROVED** [review]

File: frosting-openapi-upstream-transform/action.yml [hunk 2]
```diff
@@ -29,4 +32,11 @@ runs:
       with:
         node-version: 18
         cache: 'npm'
-    # this will be implemented with PLAT-3867 here
+
+    - name: Run install oast
```
Comment (shonteag) [inline frosting-openapi-upstream-transform/action.yml:36]: Nitpick: I still think we should call it "ghOAST" :)

File: frosting-openapi-upstream-transform/action.yml [hunk 3]
```diff
@@ -5,6 +5,9 @@ inputs:
     description: "Applies the defined global transformations to the oas document"
     required: true
     default: true
+  automation-pat:
```
Comment (donofriov) [inline frosting-openapi-upstream-transform/action.yml:8]: nit: we call this `token:` in a lot of other places but it's not a defined standard

**Review by donofriov — APPROVED** [review]

File: frosting-openapi-upstream-transform/action.yml (hunk 2 above)
Comment (jmgamboa) [reply frosting-openapi-upstream-transform/action.yml:36]: haha

File: frosting-openapi-upstream-transform/action.yml (hunk 3 above)
Comment (jmgamboa) [reply frosting-openapi-upstream-transform/action.yml:8]: yeah sure we can run with that!

File: frosting-openapi-upstream-transform/action.yml (hunk 2 above)
Comment (jmgamboa) [reply frosting-openapi-upstream-transform/action.yml:36]: there's just too many references to it at this point 😭 maybe we can unofficially call it ghoast or toast!

**Review by drojas-truelogic — APPROVED** [review]

**Pull request merged**